docs/README.md

```diff
@@ -1,6 +1,8 @@
 # Orchid
 
-Orchid is a generator which can be used to create a new CDRH API template site. The new site can either connect to the entire API's contents or filter by "type," meaning that the new site can be used for a specific collection.
+Orchid is a generator which can be used to create a new CDRH API template site.
+The new site can either connect to the entire API's contents or filter by
+"type," meaning that the new site can be used for a specific collection.
```

Comment:
I wonder if we should rephrase this, because `_type` was the old system, but since ES6 we've been using `collection` instead.  That's what the API is serving up at the endpoints, if you do `v1/collections/[collection_name]`